plugins/package-managers/cocoapods/src/main/kotlin/Lockfile.kt

```diff
@@ -48,40 +53,45 @@ internal data class Lockfile(
 }
 
 internal fun String.parseLockfile(): Lockfile {
-    val root = yamlMapper.readTree(this)
+    val root = Yaml.default.parseToYamlNode(this).yamlMap
+
+    val pods = root.yamlMap.get<YamlList>("PODS")!!.yamlList.items.map { node ->
```

Comment:
You already call `yamlMap` as part of defining `root`, so it can be dropped here.